Fix #104 for displaying Photo Library if authorization is required

newyorkpizza opened the pull request:
If FilePicker is the first code to request access to the Photo Library it must wait until after the user authorizes the app to try to fetch the assets.

dzana commented on FPPicker/Platforms/iOS/FPLocalAlbumController.m:
```diff
@@ -103,6 +103,21 @@ - (void)loadAlbumData
+        }];
+
+        // Wait until after the user has responded to the prompt.
+        return;
```
I tested and it works well, but it will be more clearly when we move this code to viewWillAppear: method.

newyorkpizza committed:
Fix #104 for displaying Photo Library if authorization is required

If FilePicker is the first code to request access to the Photo Library
it must wait until after the user authorizes the app to try to fetch
the assets.

newyorkpizza commented:
Thanks for the feedback, I moved the changes to viewWillAppear.
